backend/app/Http/Kernel.php

```diff
@@ -40,7 +40,7 @@ class Kernel extends HttpKernel
         ],
 
         'api' => [
-            \Laravel\Sanctum\Http\Middleware\EnsureFrontendRequestsAreStateful::class,
+            // \Laravel\Sanctum\Http\Middleware\EnsureFrontendRequestsAreStateful::class,
```

Comment:
I notice in you implementation you kept on commenting and uncommenting this section. Is there a reason why you commented this back?

Reply:
@kimjo77 
This caused errors when sending requests via axios on frontend (despite working on postman). I commented it to remove the errors.